### DISCUSSION TITLE
message to large

### DISCUSSION THREAD

**Discussion by hellgod77** [ZoeyVid/NPMplus#817] [Q&A] [answered]
st: "POST /upload?n=0.0014861144079907351 HTTP/2.0", host: "stream.xxxx.tech", referrer: "https://stream.xxxx.tech/"
2024/05/14 15:43:51 [alert] 1471#1471: *396 sendmsg() failed (90: Message too large) while sending frames, client: 2600:blanked:f3f1:e6e, server: [::]:443

what is this error exactly, having random issues but this is the only thing i see in the logs i don't get

**Comment by Zoey2936** [comment, marked as answer]
sorry no idea, looks like an issue of your system and/or freenginx
